### PR TITLE
[New Entry] C#: System.Math

### LINKED ISSUE #1322: [Concept Entry] C#: System.Math
### Reason/inspiration (optional)

We would like a new concept entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the built in `System.Math` Class.

The entry should go in a new file in docs/content/c-sharp/concepts/system-math/system-math.md

The entry should

- Explain what it is used for.
- Show an example of using some `System.Math` methods in code.
- End with a line introducing the list of terms that will eventually follow, like in the C++ [Math Functions](https://www.codecademy.com/resources/docs/cpp/math-functions) entry.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Zakalvve**
### Description
Closes #1322
Added the system-math.md file to a new directory system-math within existing directory c-sharp. The content covers basic use of the System.Math class that comes with C#. It also concludes by introducing a table that has currently not been created but is intended to contain a list of associated methods and properties of the Math class.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Zakalvve, I did a first review and left a few comments.

**Comment by Zakalvve** [comment]
Hi @SSwiniarski thanks for your review,

I have made the changes that you suggested. 

In addition I decided to keep the external link to learn more about .NET. You can see my reasoning in the commit. If the Codecademy docs were to grow to include .NET then the link would need to be changed.

If you decide it is not relevant then I will remove after a second review.

Thanks.

File: content/c-sharp/concepts/system-math/system-math.md
Comment (Zakalvve) [reply content/c-sharp/concepts/system-math/system-math.md]: Point referenced in commit: [Re-formatted external link to conform with guidelines.](https://github.com/Codecademy/docs/pull/1327/commits/242341a552b9513f02213f2667a32d62f0e524e8).

**Review by SSwiniarski — APPROVED** [review]
@Zakalvve, looks good for a second review.

**Comment by Zakalvve** [comment]
Are we just waiting for a second reviewer or is there something I need to do?

I see some of the checks failed. Is that something I need to correct? One says that all files should end with a new line character. Should I add that?

Sorry quite new to this!

**Comment by SSwiniarski** [comment]
@Zakalvve we're just waiting on another maintainer to give this a second review.

File: content/c-sharp/concepts/system-math/system-math.md
Comment (KTom101) [inline content/c-sharp/concepts/system-math/system-math.md:32]:
```suggestion
```

File: content/c-sharp/concepts/system-math/system-math.md
Comment (KTom101) [inline content/c-sharp/concepts/system-math/system-math.md:31]: Can we add a line printing out the variable to make it more interactive? Also Math.Pow returns a "double".
```suggestion
public class Example
{
  public static void Main(string[] args)
  {
      // Raise 4 to power of 2
      double x = Math.Pow(4,2);
      
      Console.WriteLine("x = {0}", x);
      // Output: x = 16
  }
}
```

File: content/c-sharp/concepts/system-math/system-math.md
Comment (KTom101) [inline content/c-sharp/concepts/system-math/system-math.md:43]:
```suggestion
```

File: content/c-sharp/concepts/system-math/system-math.md
Comment (KTom101) [inline content/c-sharp/concepts/system-math/system-math.md:42]:
```suggestion
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Zakalvve I've added a second review. Please reach out with any questions. Note, one main change that needs to be made is changing the title from "system.math" to "Math Functions" this just makes it a little clearer and easier to find within the topics. Changing this means the file name will need to be changed too from 
`content/c-sharp/concepts/system-math/system-math.md` ---> `content/c-sharp/concepts/math-functions/math-functions.md`

**Comment by SSwiniarski** [comment]
@Zakalvve it's been a couple of weeks, do you need assistance in implementing the feedback?

**Comment by Zakalvve** [comment]
@SSwiniarski Sorry for the delay. I have made a new commit to conform with KTom101's review. Thanks.

File: content/c-sharp/concepts/math-functions/math-functions.md
Comment (KTom101) [inline content/c-sharp/concepts/math-functions/math-functions.md:30]: Add a quick explanation of what the following example is 
```suggestion

The following example uses the `Math.Pow()` method to return the result of 4 raised to the power of 2:

```

File: content/c-sharp/concepts/math-functions/math-functions.md
Comment (KTom101) [inline content/c-sharp/concepts/math-functions/math-functions.md:48]:
```suggestion
In cases where access to System.Math is required but access to the entire System namespace is not. The following example uses an alternative syntax and returns the floor of `3/2` using the `System.Math.Floor()` method:
```

**Review by KTom101 — COMMENTED** [review]
@Zakalvve I've added a few more changes.

**Comment by Zakalvve** [comment]
@SSwiniarski @KTom101 I have made a new commit which implements the suggested changes.

**Comment by KTom101** [comment]
@Zakalvve Thank you for adding the changes. I'm noticing that some additional finals were added in the last commit. Would you be able to remove those files?

**Comment by Zakalvve** [comment]
@KTom101 Thanks for your response. I have removed the erroneous directory.

**Review by SSwiniarski — APPROVED** [review]
@KTom101 looks good to merge.

**Comment by Zakalvve** [comment]
@SSwiniarski @KTom101 Thanks for the support.

**Comment by KTom101** [comment]
@Zakalvve Thanks for contributing! You can look at the live link here: https://www.codecademy.com/resources/docs/c-sharp/math-functions

I'm noticing you're not coming up as a contributor, though. If you wish to receive credit can you make sure you've added your GitHub username to your Codecademy profile using the following steps:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

Let me know if you need any help if you decide to do so! 😁

**Pull request merged**
